src/videoTrackingHTML5.js

```diff
@@ -65,7 +65,12 @@
             eventTimestamp: currentTime,
             currentVolume: volume
         };
-        if (!seeking && hasPlayed && hasPaused == true) {
+        if (hasPaused == false && hasPlayed == false){
```

Comment:
@rezik007 this is where I have issue. If i use
`if (hasPaused && hasPlayed == false){`
it doesn't work

Reply:
@mgonera please take a look at all the compares one more time eg.
- `<something> === false`
- `<something> === true`

Doing things that way is not necessary. When JS executes it "casts" that eg. if conditions on boolean. So there is no reason to compare if <somethings> (which is already boolean) casted on boolean equals given boolean. All that strict compares could be omited. Of course there are cases when:
- our variable could have few "state-types" eg. could be undefined or true or empty object (which is not good by design but possible and worth to mention) so that strict compare if something is equal something makes sense.
- however looking at you variables as you declare them as boolean flags (true/false) you can simply omit that.